sites/dotnet-guide/contents/tutorials/resharper-essentials/inspections-quick-fixes/index.md

```diff
@@ -0,0 +1,48 @@
+---
+type: TutorialStep
+date: 2021-03-04
+title: Inspections and Quick-Fixes in ReSharper
+technologies: [.net, csharp]
+topics: [completion, inspections]
+products: [resharper]
+author: matkoch
+subtitle: Helping you write better code, faster, and automatically fixes code issues.
+thumbnail: ./thumbnail.png
+longVideo: 
+    poster: ./poster.png
+    url: https://TODO
+---
+
+### What are Inspections and Quick-Fixes?
+
+ReSharper warns you when there's a problem in your code at design time, so you see any errors, problems or possible improvements directly in the editor.
+Many of these inspections come with associated quick-fixes - an automated way of solving the issue by pressing <kbd>Alt+Enter</kbd>.
+
+Inspections are shown in the editor, using different types of underlines. The type of underline depends on the severity of an inspection.
+They can be suggestions, informational, warnings and errors.
```

Comment:
```suggestion
They can be suggestions, informational, warnings, and errors.
```

Reply:
.... hints, suggestions, warnings, or errors. Certain inspections can also be plain informational and will use individual highlighting.